src/platform/native/Core.zig

```diff
@@ -116,7 +116,7 @@ pub fn init(allocator: std.mem.Allocator, options: Options) !*Core {
     }
 
     // Print which adapter we are going to use.
-    var props: gpu.Adapter.Properties = undefined;
+    var props = std.mem.zeroes(gpu.Adapter.Properties);
```

Comment:
Why would this change? Are we sure that zero values is correct and not `.{}` (default values)?

Reply:
dawn now verifies next chain is `null`.
```zig
var props: gpu.Adapter.Properties = undefined;
props.next = null;
```
will also work

Reply:
I think we should use `var props: gpu.Adapter.Properties = .{};` because then we would get the default values (and should get `.next = null`)

Reply:
Ahh I see, if we did `.{}` default values we would need to specify `undefined` for all other fields, so `std.mem.zeroes` is easiest/cleanest. Makes sense.